src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -7652,5 +7652,26 @@ public void RecordDeclaration()
                 SymbolDisplayPartKind.Space,
                 SymbolDisplayPartKind.RecordClassName);
         }
+
+        [Fact()]
+        public void TestFunctionPointerWithoutIncludeTypesInParameterOptions()
```

Comment:
Can you add some tests for complex nested scenarios. For example, what does `delegate*<(int, string), void>` do? Or a version with tuple names?